inc/thorvg.h

```diff
@@ -1221,17 +1221,17 @@ class TVG_EXPORT SwCanvas final : public Canvas
      * while processing rendering. It internally uses one shared memory pool
      * which can be reused among the canvases in order to avoid memory overhead.
      *
-     * Thus ThorVG suggests memory pool policy to satisfy user demands,
+     * Thus ThorVG suggests using a memory pool policy to satisfy user demands,
      * if it needs to guarantee the thread-safety of the internal data access.
      *
-     * @param[in] policy Use the shared cache memory. The default value is @c true
+     * @param[in] policy The method specifying the Memory Pool behavior. The default value is @c MempoolPolicy::Default equal to @c MempoolPolicy::Shareable - a shared cache memory is used.
```

Comment:
It doesn't need to mention "equal to @c MempoolPolicy::Shareable" here.
default means default. Please don't add any internal dependency.